src/tests/ftest/posix/simul.py

```diff
@@ -0,0 +1,149 @@
+        exclude str:  comma-separated list of tests to exclude
+        If include value is set, exclude value is ignored and vice versa.
+        """
+        # Assuming all vms use the same OS
+        host_os = distro.linux_distribution()[0].lower()
+        version_os = distro.linux_distribution()[1]
+        dfuse_hosts = self.agent_managers[0].hosts
+
+        if "centos" in host_os:
+            if version_os == "8":
+                simul_dict = {"openmpi": "/usr/lib64/openmpi/bin/simul"}
+            else:
+                simul_dict = {"openmpi": "/usr/lib64/openmpi3/bin/simul"}
+        elif "suse" in host_os:
+            simul_dict = {"openmpi": "/usr/lib64/mpi/gcc/openmpi3/bin/simul"}
+        else:
+            self.log.error(host_os)
+            raise NotImplementedError
+
+        dfuse_mount_dir = self.params.get("mount_dir", '/run/dfuse/*')
+
+        if not self.pool:
+            self.log.info("Create a pool")
+            self.add_pool()
+        if not self.container:
+            self.log.info("Create container")
+            self.add_container(self.pool)
+        self.start_dfuse(dfuse_hosts, self.pool, self.container)
+        self.dfuse.check_running()
+
+        for mpi, simul in simul_dict.items():
+            # The use of MPI here is to run in parallel all simul tests
+            # on a single host.
+            load_mpi(mpi)
+            if include:
+                cmd = "{0} -vv -d {1} -i {2}".format(
+                    simul,
+                    dfuse_mount_dir,
+                    include)
+            else:
+                cmd = "{0} -vv -d {1} -e {2}".format(
+                    simul,
+                    dfuse_mount_dir,
+                    exclude)
+            try:
+                run_command(cmd)
+            except DaosTestError as exception:
+                if "FAILED in simul" in exception.args[0]:
+                    self.log.info("Expected failure")
+                    self.stop_dfuse()
+                    self.fail("Simul failed on {}".format(mpi))
+
+            self.log.info("Simul passed on %s", mpi)
+        self.stop_dfuse()
+
+    def test_posix_simul(self):
+        """Test simul.
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=posix,simul,dfuse
+        """
+        self.run_simul(exclude="9,18,30,39,40")
+
+    def test_posix_expected_failures(self):
+        """Test simul, expected failures
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=posix,simul_failure,dfuse
+        """
+        expected_failures = ["9", "18", "30", "39", "40"]
+        for test in expected_failures:
+            try:
+                self.run_simul(include=test)
+            except TestFail as exception:
+                # Only catch expected message, otherwise fail
+                if "Simul failed" in exception.args[0]:
+                    self.log.info("Expected failure")
```

Comment:
```suggestion
                    self.log.info("Expected failure: %s", test)
```
I think it's better to include the test number in this log for better readability.